test/media/time_ranges_utils_unit.js

```diff
@@ -122,7 +130,8 @@ describe('TimeRangesUtils', () => {
       it(name, () => {
         const b = createFakeBuffered(
             [{start: 10, end: 20}, {start: 30, end: 40}, {start: 50, end: 60}]);
-        expect(TimeRangesUtils.getGapIndex(b, data.time)).toBe(data.expected);
+        expect(TimeRangesUtils.getGapIndex(b, data.time, config))
```

Comment:
For these, you should pass an explicit value (i.e. not from `PlayerConfiguration`).  This makes the tests more consistent and still work if we change the default configuration value.